.gitignore

```diff
@@ -51,3 +51,16 @@ components/eamxx/docs/common/eamxx_params.md
 components/eamxx/src/python/build
 components/eamxx/src/python/build_src
 components/eamxx/src/python/dist
+
```

Comment:
@irenavankova - I don't think this should be in your PR

Reply:
Yes, I agree.  I was going to let this be because these are all sensible things to ignore but I think we'd better leave them out or make them in a separate PR.